index.bs

```diff
@@ -868,6 +868,49 @@ option. If {{UnderlyingSource/type}} is set to undefined (including via omission
     |options|["{{StreamPipeOptions/preventClose}}"],
     |options|["{{StreamPipeOptions/preventAbort}}"],
     |options|["{{StreamPipeOptions/preventCancel}}"], |signal|).
+
+ <div class="example" id="example-pipe-abortsignal">
+  An ongoing [=pipe=] operation can be stopped using an {{AbortSignal}}, as follows:
+
+  <xmp highlight="js">
+   const controller = new AbortController();
+   readable.pipeTo(writable, { signal: controller.signal });
+
+   // ... some time later ...
+   controller.abort();
+  </xmp>
+
+  (The above omits error handling for the promise returned by {{ReadableStream/pipeTo()}}.
```

Comment:
Having two sentences inside "()" looks a little odd to me. I think you could just omit the "()" here. If you like. I don't have really strong feelings about it.

Reply:
Eh, on balance I like it, as a way of deemphasizing these sentences. I'll keep it. Thanks for weighing in though.